dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -495,7 +495,7 @@ class PerfTest {
   const PerfTest.e2e(
     this.testDirectory,
     this.testTarget, {
-    this.measureCpuGpu = false,
+    this.measureCpuGpu = true,
```

Comment:
I think this shouldn't have been changed like this because e2e tests don't yet have support for CPU/GPU/memory mesurements.

Reply:
Thanks for spotting this! Previously all e2e tests were either Android or gallery. Android disabled CPU/GPU/memory anyway, and Gallery has https://github.com/flutter/flutter/issues/70438 (which gets discovered today thanks to this).

Uploaded https://github.com/flutter/flutter/pull/70439 to fix this.